server/src/instant/data/resolvers.clj

```diff
@@ -38,6 +38,19 @@
        (string/join "-")
        (str "eid-")))
 
+(defn make-attr-resolver
```

Comment:
This isn't used in the PR, but thought it would be nice. 

I found myself wanting to 'transform' the node results for an app, but didn't have the 'mapping' for ids. So, this just does the mapping for attrs